static/docs/user-guide/data-sharing/mounted-cache.md

```diff
@@ -0,0 +1,144 @@
+then effectively they will be using the same cache directory (which is mounted
+from the NAS server). So, if one of them adds something to the cache, it will
+appear automatically to the cache of all the others. As a result, no `dvc push`
+and `dvc pull` are needed to share the data, just a `dvc checkout` will be
+sufficient.
+
+> ** ❗ Caution:** Deleting data from the cache will also make it disappear from
+> the cache of the other users. So, be careful with the command `dvc gc` (which
+> cleans obsolete data from the cache) and consult the other users of the
+> project before using this command.
+
+The optimization in data management comes from using the _symlink_ cache type.
+You can find more details about it in the page of
+[Large Dataset Optimization](https://dvc.org/doc/user-guide/large-dataset-optimization).
+
+## Mounted Cache Example
+
+In this example we will see how to share data with the help of a cache directory
+that is mounted through SSHFS. We are using a SSHFS example because it is easy
+to network-mount a directory with SSHFS. However once you understand how it
+works, it should be easy to implement it for other types of network-mounted
+storages (like NFS, Samba, etc.).
+
+> For more detailed instructions check out this
+> [interactive example](https://katacoda.com/dvc/courses/examples/mounted-cache).
+
+<p align="center">
+<img src="/static/img/user-guide/data-sharing/mounted-cache.png"/>
+</p>
+
+<details>
+
+### Prerequisites: Setup the server
+
+We have to do these configurations on the SSH server:
+
+- Create accounts for each user and add them to groups for accessing the Git
+  repository and the DVC storage.
+- Create a bare git repository (for example on `/srv/project.git/`) and an empty
+  directory for the DVC cache (for example on `/srv/project.cache/`).
+- Grant users read/write access to these directories (through the groups).
+
+</details>
+
+<details>
+
+### Setup each user
+
+When we have to access a SSH server, we definitely want to generate ssh key
+pairs and setup the SSH config so that we can access the server without a
+password.
+
+Let's assume that for each user we can use the private ssh key
+`~/.ssh/dvc-server` to access the server without a password, and we have also
+added on `~/.ssh/config` lines like these:
+
+```
+Host dvc-server
+    HostName host01
+    User user1
+    IdentityFile ~/.ssh/dvc-server
+    IdentitiesOnly yes
+```
+
+Here `dvc-server` is the name or alias that we can use for our server, `host01`
+can actually be the IP or the FQDN of the server, and `user1` is the username of
+the first user on the server.
+
+</details>
+
+### Mount the DVC cache
+
+With SSHFS (and the SSH configuration on the section above), we can mount the
+remote directory to `.dvc/cache/` of the project like this:
+
+```dvc
+$ mkdir -p ~/project/.dvc/cache
+$ sshfs \
+      dvc-server:/srv/project.cache/ \
+      ~/project/.dvc/cache/
+```
+
+### Optimize data management
```

Comment:
protected mode is missed here

Reply:
> protected mode is missed here

No it is not missing, maybe you have missed it: https://dvc-org-pr-784.herokuapp.com/doc/user-guide/data-sharing/mounted-cache#optimize-data-management

Reply:
👍 probably worth mentioning why is it needed. Was mostly reading the text that's probably missed this.